fe/fe-core/src/main/java/com/starrocks/sql/analyzer/Analyzer.java

```diff
@@ -262,6 +267,22 @@ public Void visitShowCatalogsStmt(ShowCatalogsStmt statement, ConnectContext con
             return null;
         }
 
+        @Override
+        public Void visitDropFunction(DropFunctionStmt statement, ConnectContext context) {
+            DropStmtAnalyzer.analyze(statement, context);
+            return null;
+        }
+
+        @Override
+        public Void visitCreateFunction(CreateFunctionStmt statement, ConnectContext context) {
+            try {
+                statement.analyze(context, false);
```

Comment:
why we don't have to check privilege in new parser ?

Reply:
Because PrivilegeChecker has done it, and then new and old analzyer all call  
com.starrocks.analysis.CreateFunctionStmt#analyze(com.starrocks.qe.ConnectContext, boolean), just new analzyer don't have to check privilege twice.